index.ts

```diff
@@ -112,10 +149,10 @@ export function search ({
   sources = 'gn,oa,osm,wof',
   text,
   url = searchUrl
-}) {
+}: Query): Promise<Array<JSON>> {
```

Comment:
Can we store `Promise<Array<JSON>>` in its own type? Looks to be reused a few times.

Reply:
Yes we can! In 165129c2154d6b79a50f8b25ee3295a757fccfee